aqua/resources-api.aqua

```diff
@@ -262,8 +183,7 @@ func registerProvider(resource_id: ResourceId, value: string, service_id: ?strin
                   error <<- put_res.error
 
         success <- wait(successful, REPLICATION_FACTOR, DEFAULT_TIMEOUT)
-        -- HACK: workaround for #LNG-63
-        succ = BoolOp.identity(success!)
+        succ = success!
```

Comment:
Maybe move that inside `wait`?

Reply:
`wait` returns `bool`, success is stream